### PR TITLE
Update hera.yaml

### PR DESCRIPTION
**Pull request by ryanlong1004**
Bumped netcdf version from 4.7.0 -> 4.7.2

### CODE REVIEW

**Comment by rsdunlapiv** [comment]
In the [PR](https://github.com/esmf-org/esmf/pull/39), @jedwards4b said:

> PIO cannot build with netcdf-c-4.7.2, please upgrade your netCDF library 

So I don't think this will address the issue.  What versions of netcdf are available for: 
hera | gfortran 9.2.0 
hera | intel 18.0.4

**Comment by jedwards4b** [comment]
Note that the 4.7.2 version of netcdf is a particular problem.   4.7.0 works okay and you can revert to the older version if you don't have any newer one available.

**Comment by ryanlong1004** [comment]
@rsdunlapiv  Available on Hera.

netcdf
------- 
netcdf/3.6.3, 
netcdf/4.6.1, 
netcdf/4.7.0, 
netcdf/4.7.2
netcdf-hdf5

parallel
--------
netcdf-hdf5parallel/4.7.0, 
netcdf-hdf5parallel/4.7.4

pnetcdf
--------
pnetcdf/1.7.0, 
pnetcdf/1.10.0, 
pnetcdf/1.11.2

**Comment by jedwards4b** [comment]
netcdf-hdf5parallel/4.7.4 should work - can you give it a try?

**Comment by ryanlong1004** [comment]
Doing manual run on Hera now after the update.

File: config/hera.yaml [hunk 1]
```diff
@@ -1,74 +1,82 @@
...
+      hdf5: hdf5/1.10.5
+      mpi: 
+        mpiuni: 
+          module: None
+        openmpi: 
+          module: openmpi/3.1.4
+          mpi_env_vars: 
+            var1: ESMF_MPIRUN=mpirun.srun
+          pythontest: true
+      netcdf: netcdf/4.7.2
```
Comment (jedwards4b) [inline config/hera.yaml:34]: This netcdf version won't work - can you please change it?

Comment (ryanlong1004) [reply config/hera.yaml:34]: @jedwards4b updated.

https://github.com/esmf-org/esmf-test-scripts/commit/27acffca80694ecec057a24591bfb3ba4cf18de9

**Pull request merged**